notebook/Solution/04 - CNN- Univariate - Single-Step.ipynb

```diff
@@ -318,14 +309,17 @@
     "        \n",
     "        # For example Input:(N,C,Lin) Output : (N,C,Lout)\n",
     "        self.poo1 = nn.MaxPool1d(kernel_size=2)\n",
-    "        self.fc1 = nn.Linear(30, 10)\n",
+    "        \n",
+    "        self.conv2 = nn.Conv1d(in_channels = 30, out_channels = 20, kernel_size = 2)\n",
```

Comment:
it can be any number, the kernel size must be smaller than the data sequence length. For example, if your data sequence 
length is 3, your kernel size should be 2. 
 If your kernel size is larger than the data sequence length, it will throw an error due to the data is too short for the kernel to stride, unless you use padding. However, padding will make decrease the accuracy (as your network goes deeper).
Hence, kernel size =2 is the most general way to use as I increase the layer of CNN (deeper the network)

Reply:
Noted with your choice